chain/client/src/client.rs

```diff
@@ -360,6 +360,10 @@ impl Client {
         );
         let chunk_endorsement_tracker =
             Arc::new(ChunkEndorsementTracker::new(epoch_manager.clone()));
+        // Chunk validator should panic if there is a validator error in non-production chains (eg. mocket and localnet).
+        let panic_on_validation_error = config.chain_id != near_primitives::chains::MAINNET
+            && config.chain_id != near_primitives::chains::TESTNET
+            && config.chain_id != near_primitives::chains::STATELESSNET;
```

Comment:
since statelessnet nodes will be run by the community validators, I am not sure if we should panic them, so treating them as production chains.

Reply:
done.